phys/module_physics_init.F

```diff
@@ -4418,7 +4418,7 @@ SUBROUTINE mp_init(RAINNC,SNOWNC,GRAUPELNC,config_flags,restart,warm_rain,
    USE module_mp_fast_sbm
 #endif
    USE module_mp_morr_two_moment
-   USE module_mp_p3
+   USE microphy_p3
```

Comment:
@dudhia Are you ok with the module name change inside the code?

Reply:
Yes, I think we can live with it. They want to share it with another model.